src/MarginTrading.SettingsService.Services/MarketDayOffService.cs

```diff
@@ -144,8 +220,9 @@ private static bool IsBetween(DateTime currentDateTime, DateTime start, DateTime
 
                     return new[]
                     {
+                        new CompiledScheduleTimeInterval(sch, start.AddDays(-1), end.AddDays(-1)),
```

Comment:
Shouldn't we add more intervals to cover the whole week?
like +2 -2 +3 -3 etc
For instance at the day +1 interval may be covered with a highly rated single interval.. but on +2 it won't be covered, and it may work.